src/js/sjcl-custom/sjcl-ecdsa-signdeterministic.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+/*global sjcl*/
+/*eslint-disable new-cap*/
```

Comment:
Preferred disabling method is `/* eslint new-cap: [2, {newIsCapExceptions: ["bn"]}] */`